src/common/test/componentMocking.tsx

```diff
@@ -86,6 +87,30 @@ export const cleanComponentMocks = (): void => {
 };
 
 export const emptyResponseProvider: ResponseProvider = () => ({});
+export const submitButtonSelector: ElementSelector = {
+  querySelector: 'button[type="submit"]',
+};
+export const getElementAttribute = (
```

Comment:
Not sure if this function brings extra value? Could we just use element.getAttribute directly?

Reply:
Yes, there used to be check for element existence in the func, but not anymore. I'll remove the func.

Reply:
Done.